Fw/Python/src/fprime/fbuild/builder.py

```diff
@@ -344,7 +340,6 @@ def get_build_info(self, context: Path) -> dict:
             target
             for target in BUILD_TARGETS
             if isinstance(target, GlobalTarget)
-            if self.build_type in target.build_types
```

Comment:
I don't understand this removal.  Should this happen anyway?

Reply:
This would filter the output of `fprime-util info` based on the build type. Since I don't see a major use case for filtering based on build type, I figured it was easier to remove this functionality than to update it to work with the other code changes